app/dependencies.py

```diff
@@ -17,11 +18,12 @@ def __init__(self):
         # general config
         self.config_base_path = os.environ.get("CONFIG_PATH") or "/tmp"
         self.smd_topic = os.environ.get("SMD_TOPIC") or "rddl/SMD/#"
-        self.path_to_smart_meter_config = build_config_path(self.config_base_path, FILE_SMART_METER_CONFIG)
         self.path_to_mqtt_config = build_config_path(self.config_base_path, FILE_MQTT_CONFIG)
+        self.path_to_smart_meter_config = build_config_path(self.config_base_path, FILE_SMART_METER_CONFIG)
+        self.path_to_smart_meter_mqtt_config = build_config_path(self.config_base_path, FILE_SMART_METER_MQTT_CONFIG)
         self.trust_wallet_port = os.environ.get("TRUST_WALLET_PORT") or "/dev/ttyACM0"
-        self.notarize_interval = int(os.environ.get("NOTARIZE_INTERVAL") or 60)
-        self.rddl_network_mode = os.environ.get("RDDL_NETWORK_MODE") or "mainnet"
+        self.notarize_interval = int(os.environ.get("NOTARIZE_INTERVAL") or 2)
```

Comment:
aren't these test values? shouldn't we set them back to 60 and mainnet?

Reply:
You are right i overlooked that one!